docs/userguides/accounts.md

```diff
@@ -234,6 +234,23 @@ message = encode_defunct(text="Hello Apes!")
 signature = account.sign_message(message)
 ```
 
+## Signed Message
+
+Get the signed message signature as well as message hash signed by the account
+
+```py
+from ape.types.signatures import recover_signer
```

Comment:
Is this import needed? `recover_signer` is imported but not even used in his example.

Reply:
The final docs are not updated. It's a draft & waiting for dog

Reply:
Waiting for the busiest person at ApeWorX instead of working with me who is directly trying to help you now is not a good strategy.

Reply:
Don't get me wrong I could use all the help I get 😅

Reply:
I may be the first person in the world to get a thumbs down in a pr conversation 😶